misk-jdbc/src/main/kotlin/misk/jdbc/JdbcModule.kt

```diff
@@ -0,0 +1,177 @@
+package misk.jdbc
+
+import io.opentracing.Tracer
+import io.prometheus.client.CollectorRegistry
+import misk.ServiceModule
+import misk.database.StartDatabaseService
+import misk.environment.Environment
+import misk.healthchecks.HealthCheck
+import misk.inject.KAbstractModule
+import misk.inject.asSingleton
+import misk.inject.keyOf
+import misk.inject.setOfType
+import misk.inject.toKey
+import misk.resources.ResourceLoader
+import javax.inject.Inject
+import javax.inject.Provider
+import javax.sql.DataSource
+import kotlin.reflect.KClass
+
+/**
+ * Binds database connectivity for a qualified data source. This binds the following public types:
+ *
+ *  * @Qualifier [misk.jdbc.DataSourceConfig]
+ *
+ * This also registers services to connect to the database ([SessionFactoryService]) and to verify
+ * that the schema is up-to-date ([SchemaMigratorService]).
+ */
```

Comment:
if you move this documentation to immediately precede JdbcModule, then it’ll be ‘attached’ to that in Javadoc and IDE integration. As-is it’s attached to `MAX_MAX_ROWS`.